ddtrace/contrib/celery/utils.py

```diff
@@ -33,27 +34,37 @@
 )
 
 
+def should_skip_context_value(key, value):
```

Comment:
do we need this helper? the original code seems to be doing the same thing?

Reply:
I lean towards more helpers so I can avoid large functions, but happy to conform to whatever is more standard for the codebase.

Reply:
no strong preference from me 🙂